rfcs/README.md

```diff
@@ -0,0 +1,62 @@
+# Gestalt RFCs
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put
+through a bit of a design process and produce a consensus among the Gestalt team.
+
+The "RFC" (request for comments) process is intended to provide a
+consistent and controlled path for new features to enter the design system.
+
+Furthermore, Gestalt's RFC repository is a source of institutional knowledge around Gestalt's decision process and development.
+
+## When you need to follow this process
+
+You need to follow this process if you intend to make "substantial"
+changes to any of the Gestalt packages or its documentation.
+
+What constitutes a "substantial" change may include the following:
+
+- A new Gestalt building-block component that solves complex library issues (such as [ScrollBoundaryContainer](https://gestalt.pinterest.systems/scrollboundarycontainer) or [Z-Index classes](https://gestalt.pinterest.systems/zindex_classes)) and/or adds new developing patterns to Gestalt users (such as [Flex](https://gestalt.pinterest.systems/flex) or [TapArea](https://gestalt.pinterest.systems/taparea)).
+- A new Gestalt utility component (such as [OnLinkNavigationProvider](https://gestalt.pinterest.systems/onlinknavigationprovider) or [useReducedMotion](https://gestalt.pinterest.systems/usereducedmotion)).
+- A new feature that creates new API surface area (such as [OnLinkNavigationProvider](https://gestalt.pinterest.systems/onlinknavigationprovider) and `dangerouslyDisableOnNavigation`).
+- The introduction of new idiomatic usage, conventions, or patterns (such as [Gestalt design tokens](https://gestalt.pinterest.systems/design_tokens), [boint units](https://gestalt.pinterest.systems/faq#Component-usage), or subcomponents modularity patterns like [Table](https://gestalt.netlify.app/table#Props) or [Dropdown](https://gestalt.netlify.app/dropdown) subcomponents).
+- Technology migrations (such as the [migration to Next.js](https://github.com/pinterest/gestalt/pull/1642))
+
+Some changes do not require an RFC:
+
+- Rephrasing, reorganizing or refactoring
+- New general components
+- Additions that strictly improve objective, numerical quality
+  criteria (speedup, better browser support)
+
+If you submit a pull request to implement a new feature without going
+through the RFC process, it may be closed with a polite request to
+submit an RFC first.
+
+## How to submit an RFC
+
+1. Create a directory inside this `rfcs` directory. The directory name should begin with the year and include a meaningful description, such as `rfcs/2022-onInteraction-new-feature`.
```

Comment:
Could we include the month as well, or do you think that's too much? If this scales to include a number of RFCs (and as we migrate over existing docs from other sources) I think the month might help establish a timeline a bit better. Totally open to be talked out of that opinion though.

Reply:
DONE